### DISCUSSION TITLE
Add multiple days to the same event without making it recurring

### DISCUSSION THREAD

**Discussion by zsviczian** [davish/obsidian-full-calendar#343] [Feature Requests]
Say there is a project that you work on, on different days at different times (i.e. not a recursive event), but you don't want to create separate files. Would be great if I could add dates and times to an event to display them on the calendar at different times.

One way you could do this is by allowing multiple values for startTime, endTime, and date. Like this:

```markdown
---
title: demo project
allDay: false
startTime: ["10:00", "13:30", "09:30"]
endTime: ["12:30", "18:00", "11:15"]
date: ["2022-02-20", "2022-02-21", "2022-03-01"]
---
```

**Comment by kalmir** [comment]
I would love that as well, to be used when time-blocking (as opposed to scheduling meetings and such). Zsviczian described how that could look like inside the "task or project note". It would be cool if we could update the values by clicking on the calendar and creating an event with the same name as another, already existing, one. In this case no event note would be created, instead the existing note would be updated.

And thanks for the plugin, it is already quite usable and something I was hoping for.

**Comment by zsviczian** [comment]
> It would be cool if we could update the values by clicking on the calendar and creating an event with the same name as another, already existing, one. 

I 100% agree, however I would leave this for a next phase. The MVP solution is to handle extended metadata from the task or project note. UI magic can come next...
